.gitignore

```diff
@@ -7,3 +7,4 @@ _obj
 Dockerfile
 .status.*
 *.bck
+CHANGELOG.md
```

Comment:
Is the idea here that the changelog will _only_ be present in release artifacts? It would be nice if we could find a way to keep a copy in the repo itself as well IMHO.

Reply:
This file was intended to use to bypass the issue with `gren release`. So far the release notes are stored in the GitHub releases. 
We can enable the `CHANGELOG.md` within this repo in a follow up, what do you think?

Reply:
Fine to do in a follow-up. Thanks.